app/controllers/tag_controller.rb

```diff
@@ -19,31 +25,39 @@ def index
       .where('community_tags.date > ?', (DateTime.now - 1.month).to_i)
       .where("name LIKE :prefix", prefix: "#{prefix}%")
       .group(:name)
-      .order('count DESC')
+      .order(order_string)
       .paginate(page: params[:page], per_page: 24)
     elsif @toggle == "uses"
     @tags = Tag.joins(:node_tag, :node)
       .select('node.nid, node.status, term_data.*, community_tags.*')
       .where('node.status = ?', 1)
       .where('community_tags.date > ?', (DateTime.now - 1.month).to_i)
       .group(:name)
-      .order('count DESC')
+      .order(order_string)
       .paginate(page: params[:page], per_page: 24)
     elsif @toggle == "name"
+    order_string = "name DESC"
```

Comment:
Yeah, if you do what I was saying, creating a function for defining `order_string`, this would be come more DRY. 🎉